src/main/java/net/mcreator/ui/minecraft/states/PropertyData.java

```diff
@@ -244,7 +244,7 @@ public NumberType(String name, double min, double max) {
 		}
 
 		@Override public final String toString(Object value) {
-			return df.format((double) (Double) value);
+			return df.format(value);

```

Comment:
Why was this here in the first place?

And what changed that this is no longer needed?

(additional question: is this related to the PR, and if yes, how?)